.travis.yml

```diff
@@ -0,0 +1,37 @@
+dist: trusty
+sudo: true
+
+matrix:
+  include:
+    - os: osx
+      env: TARGET=x86_64-apple-darwin RID=osx-x64
+    - os: linux
+      env: TARGET=x86_64-unknown-linux-gnu RID=linux-x64
+
+install:
+- if [ "$TRAVIS_OS_NAME" = "osx" ]; then brew install glib cairo libexif; fi
+- if [ "$TRAVIS_OS_NAME" = "linux" ]; then sudo apt-get install -y libgif-dev xvfb; fi
+- if [ "$TRAVIS_OS_NAME" = "linux" ]; then sudo apt-get install -y git autoconf libtool automake build-essential mono-devel gettext cmake; fi
```

Comment:
you can remove `mono-devel`. Mono will download a required minimal bootstrapper during build.

This also allows us to make sure we don't have any distro-provided libgdiplus on the system